debian/debathena-pyhesiodfs.postinst

```diff
@@ -25,8 +25,17 @@ set -e
 
 case "$1" in
     configure)
+
+    # Adding pyhesiodfs only if the local user group fuse exists.
+    # The group fuse no longer exists in jessie.
+    # If the group fuse is needed, it should have been added when the package 
+    # fuse, a dependency, was configured, before this script is run.
+    if grep "^fuse:" /etc/group > /dev/null 2>&1; then
```

Comment:
`if grep -q "^fuse:" /etc/group; then`

(It’s always better to avoid suppressing more errors than we’re expecting.)